{{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,148 @@
+---
+name: 🐛  Bug Report
+description: Create a report to help us improve
+labels: ["bug", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug in {{ cookiecutter.project_name }}!**
+
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+
+      [issue search]: https://github.com/osl-incubator/cookiecutter-python/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+- type: textarea
+  attributes:
+    label: Summary
+    description: Explain the problem briefly below.
+    placeholder: >-
+      When I try to do X with {{ cookiecutter.project_name }} and the following workspace, Y breaks or
```

Comment:
This is a hunch, but I think using `{{ }}` in YAML might bring up some difficulties, given that `{{ }}` is used in YAML natively too.